data/strings/strings.txt

```diff
@@ -21205,3 +21205,171 @@
     hi = बाद में मैनुअल रूप से अपडेट करें
     sk = Manuálne aktualizovať neskôr
     sw = Sasisha mwenyewe baadaye
+
+  [placepage_delete_track_button]
+    comment = Delete track button on track edit screen
+    tags = ios
+    en = Delete Track
+    ru = Удалить трек
+    bg = Delete Track
```

Comment:
Please delete all other translations, they should be loaded from the default en language automatically.

Reply:
I've added translations for "track_title" by google translate, but I'm not sure it is correct at all.

for all other new keys I've removed all non-english and non-russian keys

Reply:
I think that correct english text is better than wrong, but localized :)

Reply:
There are higher chances that someone will fix incorrect text :) From 2011, we always used Google Translate and had not so many errors. Anyway, en should be used by default without writing it explicitly.

Reply:
Ok, I'll translate other keys with google.
Btw, I see that the number of supported language codes differs from key to key (some of them have more than others).
What is the full list of supported localizations? (for example I don't see Seribian there)

Reply:
Can I use script from #941 and the updated list of languages from the script? 
I've already tried old version (I've installed translate-shell from homebrew), all worked well, but looks like the list of languages was changed.

Reply:
And maybe there is a reason to change script output format (to let it match strings.txt format), e.g.
current:
```
cs:Dráha
da:Spore
de:Schiene
```

new
```
cs = Dráha
da = Spore
de =Schiene
```

Reply:
>I see that the number of supported language codes differs from key to key (some of them have more than others).
What is the full list of supported localizations? (for example I don't see Seribian there)

We are trying to fix this mess and move all localization work to Weblate.  Please use your best efforts to fix add new strings and we will try to fix the localization process in general.

Reply:
@ivkosh the output from the script was designed for categories.txt file. The more languages you put is the better :)